src/main/java/com/ibm/watson/litelinks/ServiceProperties.java

```diff
@@ -39,6 +45,35 @@ private ThriftConnProp() {}
     public static final String METH_INFO_PREFIX = "methodinfo.";
     public static final String APP_METADATA_PREFIX = "app.";
 
+    public static final Map<String, String> SERVICE_CLASS_ALIASES;
+
+    static {
+        String serviceClassAliasConfig = System.getProperty(LitelinksSystemPropNames.SERVICE_CLASS_ALIASES);
+        if (serviceClassAliasConfig == null) {
+            serviceClassAliasConfig = System.getenv(LitelinksEnvVariableNames.SERVICE_CLASS_ALIASES);
+        }
```

Comment:
aww, no fun library like [owner](http://owner.aeonbits.org/) to do this for you? :(

Reply:
:) Generally try to minimize dependencies, don't think it's worth adding a new one just for something like this.